src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/jobMain.groovy

```diff
@@ -1,6 +1,13 @@
 package com.axis.system.jenkins.plugins.downstream.yabv.BuildFlowAction
 
-if (my.hasUpstreamOrDownstreamBuilds()) {
+import hudson.model.Run
+
+Run lastCompletedBuild = my.target?.parent?.lastCompletedBuild
+Run lastBuild = my.target?.parent?.lastBuild
+
+if (my.hasUpstreamOrDownstreamBuilds(my.target) ||
```

Comment:
Super nit, now that hasUpstreamOrDownstreamBuilds is static it should not be referenced via object reference. It should be static class reference